lib/trino-parquet/src/main/java/io/trino/parquet/reader/decoders/PlainByteArrayDecoders.java

```diff
@@ -168,6 +168,36 @@ public void skip(int n)
         {
             skipPlainValues(input, n);
         }
+
+        /**
+         * Create one big slice of data and add it to the output buffer since buffer size is known.
+         * Specialized for the case of strings with no truncation
+         */
+        private static Slice createOutputBuffer(int[] offsets, int offset, int length, Slice inputSlice, int outputBufferSize)
```

Comment:
nit: this version of `createOutputBuffer` should probably be below the original one

Reply:
I wanted to keep it as member of inner class BinaryPlainValueDecoder to avoid it getting used from the other decoders